dev/archery/archery/release/core.py

```diff
@@ -250,15 +323,9 @@ def __new__(self, version, jira=None, repo=None):
 
         return super().__new__(klass)
 
-    def __init__(self, version, jira, repo):
-        if jira is None:
-            jira = Jira()
-        elif isinstance(jira, str):
-            jira = Jira(jira)
-        elif not isinstance(jira, (Jira, CachedJira)):
-            raise TypeError("`jira` argument must be a server url or a valid "
-                            "Jira instance")
-
+    def __init__(self, version, repo, github_token=None, issue_tracker=None):
+        if not issue_tracker:
+            issue_tracker = IssueTracker(github_token=github_token)
```

Comment:
How about requiring `issue_tracker` instead of accepting optional `github_token`?